sebs/utils.py

```diff
@@ -168,21 +171,26 @@ class LoggingBase:
     def __init__(self):
         uuid_name = str(uuid.uuid4())[0:4]
         if hasattr(self, "typename"):
-            self.logging = logging.getLogger(f"{self.typename()}-{uuid_name}")
+            self._logging = logging.getLogger(f"{self.typename()}-{uuid_name}")
         else:
-            self.logging = logging.getLogger(f"{self.__class__.__name__}-{uuid_name}")
-        self.logging.setLevel(logging.INFO)
+            self._logging = logging.getLogger(f"{self.__class__.__name__}-{uuid_name}")
+        self._logging.setLevel(logging.INFO)
+        self.colored_printer = ColoredPrinter(self._logging)
 
     @property
     def logging_handlers(self) -> LoggingHandlers:
         return self._logging_handlers
+    
+    @property
```

Comment:
Unfortunately, `mypy` is broken here - it requires each property getter and setter to be next to each other to work properly :-( Not a fault on your side, but we need to move `logging` such that `logging_handlers` `@property` and `setter` are side by side.

https://github.com/python/mypy/issues/1465

Reply:
Fixed.

Reply:
Got it. I have it fixed.